backend-dev/Decisions.md

```diff
@@ -0,0 +1,32 @@
+# Decisions
+
+All questions and concerns that have come up will be documented here along with their answers.
+
+### What, if anything, should we do to safeguard the public-facing data on this site?
+
+The data on this site was public to begin with.  Scraped from other public sites, or given to us to use publicly with consent.
```

Comment:
This is a sentence fragment. Suggestion:

"We compiled public data. and we were given data with consent for public use."

Something like that? We can mention scraping if we want. IANAL, but I don't think it's that important to clarify scraping vs manual curation. (I'm also attempting to stay away from a passive voice, though that's less crucial.)